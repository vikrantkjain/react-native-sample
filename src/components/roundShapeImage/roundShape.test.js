import renderer from 'react-test-renderer';
import RoundShapeImage from './index';

test('Test round image', () => {
  const tree = renderer
    .create(
      <RoundShapeImage
        height={100}
        imageUrl={{uri: 'https://picsum.photos/200/300'}}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
