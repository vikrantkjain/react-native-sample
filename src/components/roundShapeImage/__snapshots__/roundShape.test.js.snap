// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test round image 1`] = `
<View
  style={
    {
      "alignSelf": "center",
      "margin": 10,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": true,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      [
        {
          "alignSelf": "center",
          "borderColor": "rgba(245,192,32,1)",
          "borderWidth": 5,
          "justifyContent": "center",
          "overflow": "hidden",
        },
        {
          "borderRadius": 50,
          "height": 100,
          "width": 100,
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "overflow": "hidden",
          },
          {
            "alignSelf": "center",
            "height": "100%",
            "width": "100%",
          },
        ]
      }
    >
      <FastImageView
        defaultSource={null}
        resizeMode="stretch"
        source={
          {
            "uri": "https://picsum.photos/200/300",
          }
        }
        style={
          {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      />
    </View>
  </View>
</View>
`;
